src/fsw/FCCode/GomspaceController.cpp

```diff
@@ -74,6 +74,9 @@ GomspaceController::GomspaceController(StateFieldRegistry &registry, unsigned in
     heater_sr(),
     heater_f("gomspace.heater", heater_sr),
 
+    period_sr(),
+    period_f("gomspace.period", period_sr),
```

Comment:
```suggestion
    period_f("gomspace.powercycle_period", period_sr),
```
Just to be more descriptive with the state field name.